### ISSUE TITLE
Indicator as widget without ViewPager

### ISSUE THREAD

**Issue by lucalooz** [JakeWharton/ViewPagerIndicator#207]
Hi,
i think it's a feature request, i want to use this indicator as a picker widget without a ViewPager.
Is it possible?
Thank you

**Comment by SimonVT** [comment]
No, this is not possible and it is outside the scope of this library. It would also be overkill to use this library for a static indicator, that you could easily make with a simple layout.
